modal/cli/import_refs.py

```diff
@@ -1,8 +1,10 @@
 # Copyright Modal Labs 2023
-"""Utilities for CLI references to Modal entities
+"""Load or import Python modules from the CLI.
 
 For example, the function reference of `modal run some_file.py::stub.foo_func`
-or the stub lookup of `modal deploy some_file.py`
+or the stub lookup of `modal deploy some_file.py`.
+
+These functions are only called by the Modal CLI, not in tasks.
```

Comment:
If I'm following the code properly, we also have `modal.serving.serve_stub` which uses some of these functions. That's pretty convoluted though!

Reply:
Nice find. I think `modal.serving` isn't public API and we should mention this!

Reply:
Not public API or not *intended* to be public API? ;)

Reply:
Not intended to be public API!